Directory.Build.props

```diff
@@ -86,22 +86,16 @@
     <IsPrerelease>true</IsPrerelease>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <!-- Need to set ProjectAssetsFile to something so it doesn't get set to the Tools.proj assets file when we import the generated props -->
-    <ProjectAssetsFile Condition="'$(ProjectAssetsFile)' == ''">unused</ProjectAssetsFile>
-    <!-- Nuget will normally disable all generated imports as part of a restore to avoid 
-         non-determinism where first restore changes a subsequent restore,
-         but we need these since they define properties (like TargetFramework) needed by restore.
-         In this case we garuntee this is ok since these specific props are not restored by this project's restore. -->
-    <_excludeRestorePackageImports>$(ExcludeRestorePackageImports)</_excludeRestorePackageImports>
-    <ExcludeRestorePackageImports>false</ExcludeRestorePackageImports>
-  </PropertyGroup>
-  <Import Project="$(ToolSetCommonDirectory)Tools.proj.nuget.g.props" Condition="Exists('$(ToolSetCommonDirectory)Tools.proj.nuget.g.props')" />
-  <PropertyGroup>
-    <ProjectAssetsFile Condition="'$(ProjectAssetsFile)' == 'unused'" />
-    <ExcludeRestorePackageImports>$(_excludeRestorePackageImports)</ExcludeRestorePackageImports>
+  <Import Sdk="Microsoft.DotNet.Build.Tasks.Configuration" Project="Sdk.props" />
+  <Import Sdk="Microsoft.DotNet.CoreFxTesting" Project="Sdk.props" />
```

Comment:
What's going to happen for source build?

Reply:
I removed the sourcebuild exclusion in the CoreFxTesting project.